# https://bugs.gentoo.org/915694
# Not needed for newer branches, that greatly refactored the code
--- a/src/gromacs/utility/flags.h~	2023-01-31 12:45:46.000000000 +0100
+++ b/src/gromacs/utility/flags.h	2023-10-16 10:15:24.087340444 +0200
@@ -42,7 +42,7 @@
  */
 #ifndef GMX_UTILITY_FLAGS_H
 #define GMX_UTILITY_FLAGS_H
-
+#include <cstdint>
 namespace gmx
 {
